Dockerfile

```diff
@@ -7,6 +7,7 @@ ENV GOOS=linux \
 
 # this value changes in ./bin/build
 ARG TAG="dev"
+ARG VersionSuffix="pre-release"
```

Comment:
1. Why is it `pre-release` and not `dev`?
2. It should be all-caps
3. I would call it `TAG_SUFFIX` as i say here: https://github.com/cyberark/conjur-authn-k8s-client/pull/224/files#r591725710